src/static/helpers/fetchHelper.test.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import nock from 'nock';
+import {Response} from 'node-fetch';
+import * as environment from './environment';
+import ClientConfig from '../clientConfig';
+import {runFetchHelper} from './fetchHelper';
+
+describe('runFetchHelper', () => {
```

Comment:
The fetch helper accepts a client config and uses it to assign headers amongst other things right? Do we want to test the scenario where we are overwriting a header value, which valve takes precedence? Lets lock it down with a test